src/main/java/com/lorepo/icplayer/client/module/text/TextPrintable.java

```diff
@@ -145,19 +192,31 @@ private String makePrintableDropdowns(String parsedText, boolean showAnswers) {
 			NodeList<Element> options = select.getElementsByTagName("option");
 			
 			String values = "";
-			for (int j = 0; j < options.getLength(); j++) {
-				Element option = options.getItem(j);
-				String value = option.getInnerText();
-				if (!value.equals("---")) {
-					if (showAnswers) {
-						InlineChoiceInfo choiceInfo = model.getChoiceInfos().get(i);
-						if (choiceInfo.getAnswer().equals(value)) {
-							value = "<u>" + value + "</u>";
+			JavaScriptUtils.log("C");
+			if (this.userAnswers != null && showAnswers) {

```

Comment:
Ten sam problem co w tickecie: https://learneticsa.assembla.com/spaces/lorepo/tickets/realtime_cardwall?ticket=7903 czyli brak testow, przerzucenie kreacji htmla nie tam gdzie trzeba, prosze sie skonsultujcie i poprawcie to - nie chce mi sie przepisywac tych samych komentarzy